packages/preview/whalogen-0.1.0/README.md

```diff
@@ -0,0 +1,21 @@
+# whalogen
+
+whalogen is a library for typsetting chemical formulas with Typst inspired by mhchem.
+
+
+
+## Examples
+
+![](gallery/example.png)
+
+```typst
+#import "@local/whalogen:0.1.0": ce
+
+#set page(width: auto, height: auto)
+
+$
+#ce("HCl + H2O -> H3O+ + Cl-")
+$
```

Comment:
This should use the `@preview` import. I'd also suggest keeping other formatting as minimal as possible (the example also works well without the page settings). The newline math dollar style is also not really the intended style, but that's up to you ;)

```suggestion
#import "@preview/whalogen:0.1.0": ce

$ #ce("HCl + H2O -> H3O+ + Cl-") $
```